collectors/CMPCollector.js

```diff
@@ -32,28 +46,55 @@ function isIgnoredEvalError(e) {
     );
 }
 
-class CMPCollector extends BaseCollector {
+// TODO: check for false positive detections per pattern
+const DETECT_PATTERNS = [
+    /accept cookies/i,
+    /accept all/i,
+    /reject all/i,
+    /only necessary cookies/i, // "only necessary" is probably too broad
+    /by clicking.*(accept|agree|allow)/i,
+    /by continuing/i,
+    /we (use|serve)( optional)? cookies/i,
+    /we are using cookies/i,
+    /use of cookies/i,
+    /(this|our) (web)?site.*cookies/i,
+    /cookies (and|or) .* technologies/i,
+    /such as cookies/i,
+    /read more about.*cookies/i,
+    /consent to.*cookies/i,
+    /we and our partners.*cookies/i,
+    /we.*store.*information.*such as.*cookies/i,
+    /store and\/or access information.*on a device/i,
+    /personalised ads and content, ad and content measurement/i,
 
+    // these below cause many false positives
+    // /cookies? settings/i,
+    // /cookies? preferences/i,
```

Comment:
Is it just a stylistic concern? It may be very tempting to add these phrases again: many actual popups have these buttons, but the problem is that these buttons also appear in site footers a lot. So I wanted to keep them here as a warning for future.

Reply:
Ah I see, so this is not just dead code but a warning for the future. Let's chenge the description to be more explicit e.g. like you said: "it might be tempting to add patterns like below, but from testing we know that …"